src/parser/jobs/ast/modules/Lightspeed.js

```diff
@@ -125,17 +125,22 @@ export default class LIGHTSPEED extends Module {
 				.map(cast => getDataBy(ACTIONS, 'id', cast.ability.guid))
 				.filter(action => action && action.onGcd)
 			const numGcds = gcdActions.length
-			const mpSavings = gcdActions
-				.reduce((totalSavings, action) => action.mpCost / 2 + totalSavings, 0)
-			// TODO: Use mpCostFactor instead of mpCost to be level agnostic
+
+			// pre-5.3 mp savings calcuation
+			let mpSavingsDisplay
+			if (this.parser.patch.before('5.3')) {
+				const mpSavings = gcdActions.reduce((totalSavings, action) => action.mpCost / 2 + totalSavings, 0)
+
+				mpSavingsDisplay = <>&nbsp;-&nbsp; {mpSavings} <Trans id="ast.lightspeed.rotation.mp-saved">MP saved</Trans></>
+			}
 
 			return {
 				key: lightspeed.start,
 				title: {
 					content: <>
 						{this.parser.formatTimestamp(lightspeed.start)}
 						&nbsp;-&nbsp;<Trans id="ast.lightspeed.rotation.gcd"><Plural value={numGcds} one="# GCD" other="# GCDs"/></Trans>
-						&nbsp;-&nbsp;{mpSavings} <Trans id="ast.lightspeed.rotation.mp-saved">MP saved</Trans>
+						{mpSavingsDisplay}
 					</>,
 				},
```

Comment:
Disregarding how much I hate that snippet, this might be cleaner to just set and run the patch conditional inside the content, like `{this.parser.patch.before('5.3') && mpSavingsDisplay}` I think will do it. Prob need to check this in browser but it'll let you save the const and avoid a useless reassignment.

Reply:
I haet this snippet too :(
your point is well received thx